### PR TITLE
adding productId as a property

### PR DESCRIPTION
**Pull request by mattaebersold**
@weotch I was having trouble seeing this locally when linked (slack) but it's such a simple change I created the request anyways

### CODE REVIEW

**Comment by mattaebersold** [comment]
ok @weotch it works as expected: 
image

File: index.coffee [hunk 1]
```diff
@@ -361,8 +363,11 @@ class StorefrontError extends Error
 # becomes 34641879105581
 # https://regex101.com/r/3FIplL/1
 getShopifyId = (id) ->
+	console.log 'id', id
```
Comment (weotch) [inline index.coffee:366]: Remove console logs

```suggestion
```

File: index.coffee [hunk 2]
```diff
@@ -361,8 +363,11 @@ class StorefrontError extends Error
...
 	return id if String(id).match /^\d+$/ # Already simple id
 	id = atob id unless id.match /^gid:\/\// # De-base64
+
+	console.log 'id', id
```
Comment (weotch) [inline index.coffee:370]:
```suggestion
```

**Review by weotch — CHANGES_REQUESTED** [review]
Can you recompile and push up after making these changes?

**Comment by mattaebersold** [comment]
@weotch shoot, yea sorry it's updated

**Review by weotch — APPROVED** [review]

**Comment by weotch** [comment]
@mattaebersold this is tagged as `0.3.0` now.  Thanks for the add!

**Pull request merged**